go.mod

```diff
@@ -30,8 +32,10 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.7.1
 	github.com/stretchr/testify v1.6.0
-	golang.org/x/net v0.0.0-20200528225125-3c3fba18258b
```

Comment:
Since we only rename variables and packages in this task, I don't expect `go.mod` to change. I think it is changed accidentally. Please revert it if it was not intended.

Reply:
[Reverted](https://github.com/hyperledger-labs/perun-node/pull/135/commits/d7c5500a7b2664abeae8f449eddbbd7d163a6b49)